lib/sage/executor.ex:395: Expression produces a value of type 'ok' | {'error',_}, but this value is unmatched
lib/sage/executor.ex:462: Expression produces a value of type [any()], but this value is unmatched
lib/sage/exceptions.ex:48: Type specification 'Elixir.Sage.DuplicateOperationError':exception(elixir:keyword()) -> 'Elixir.Exception':t()
